### ISSUE TITLE
Unrelated cavatica test failure

### ISSUE THREAD

**Issue by fiendish** [kids-first/kf-api-study-creator#530]
https://github.com/kids-first/kf-api-study-creator/pull/527 had a seemingly unrelated test failure on Circle. Starting the tests over subsequently passed.

Partial failure output:

```
    def test_workflows(db, settings, mocker, clients, mock_post):
        ...
        setup_cavatica.assert_called_with(
            setup_cavatica_task,
            Study.objects.first().kf_id,
            ["bwa_mem"],
            User.objects.first().sub,
>           depends_on=None,
        )
E       AssertionError: Expected call: enqueue(<function setup_cavatica_task at 0x7fdcf9494050>, 'SD_6HET65MK', ['bwa_mem'], '2d2afc9b-1f4d-4525-aebb-3e487d004f19', depends_on=None)
E       Actual call: enqueue(<function setup_cavatica_task at 0x7fdcf9494050>, 'SD_6HET65MK', ['bwa_mem'], 'feb49687-2a4d-4069-b3ef-9ee27474cbfb', depends_on=None)
E       
E       pytest introspection follows:
E       
E       Args:
E       assert (<function se...9ee27474cbfb') == (<function set...3e487d004f19')
E         At index 3 diff: 'feb49687-2a4d-4069-b3ef-9ee27474cbfb' != '2d2afc9b-1f4d-4525-aebb-3e487d004f19'
E         Use -v to get the full diff

tests/studies/test_create_study.py:302: AssertionError
```

**Comment by gsantia** [comment]
I've gotten this too before!

**Comment by dankolbman** [comment]
Should be solved by #544
